lib/utils/error-message.js

```diff
@@ -268,7 +268,6 @@ function errorMessage (er) {
       detail.push([
         'enoent',
         [
-          er.message,
```

Comment:
It's already included in the short message and we never print the detail without the short message.